### ISSUE TITLE
Enable HTTPS?

### ISSUE THREAD

**Issue by Eimuex** [iilunin/crypto-bot#17]
Is there a way to access it over HTTPS? Thank you

**Comment by iilunin** [comment]
No, there's no HTTPS support

**Comment by Eimuex** [comment]
Thanks for the response. One more question, can I run it behind jwilder/nginx-proxy maybe?

**Comment by iilunin** [comment]
Yes, I don't think there would be any issues. You'll need to enable SSL termination after the traffic passes nginx

**Comment by Eimuex** [comment]
Thanks a lot
